ash/src/extensions/experimental/amd.rs

```diff
@@ -40,6 +40,10 @@ vk_bitflags_wrapped!(
     Flags
 );
 impl fmt::Debug for GpaSqShaderStageFlags {
+    // ignore clippy::use_self false positives
+    // changing GpaSqShaderStageFlags::PS.0 to Self::PS.0 as suggested by clippy generates:
+    // error[E0401]: can't use generic parameters from outer function
+    #![allow(clippy::use_self)]
```

Comment:
Yeah, that's a bit unfortunate. I don't think you intended to use `#!` to apply it to the surrounding `impl fmt::Debug for GpaSqShaderStageFlags` scope? Unfortunately it doesn't seem like `#[allow(clippy::use_self)]` can be applied to just the statement or the `fn fmt` function, it has to be applied to the `impl` block as a whole.

```suggestion
// ignore clippy::use_self false positives
// changing GpaSqShaderStageFlags::PS.0 to Self::PS.0 as suggested by clippy generates:
// error[E0401]: can't use generic parameters from outer function
#[allow(clippy::use_self)]
impl fmt::Debug for GpaSqShaderStageFlags {
```

Reply:
Done.